src/StimulusBundle/src/Form/Extension/FormTypeExtension.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Form\Extension;
+
+use Symfony\Component\Form\AbstractTypeExtension;
+use Symfony\Component\Form\Extension\Core\Type\FormType;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\UX\StimulusBundle\Dto\StimulusAttributes;
+use Twig\Environment;
+use Twig\Loader\ArrayLoader;
+
+class FormTypeExtension extends AbstractTypeExtension
+{
+    private StimulusAttributes $stimulusAttributes;
```

Comment:
This is temporary right ?

I think we won't need it and use a new object but if we don't, you can inject the StimulusHelper here